src/render_tasks/d3d12_build_acceleration_structures.hpp

```diff
@@ -298,6 +434,8 @@ namespace wr
 
 			data.out_materials_require_update = false;
 
+			data.current_frame_index = n_render_system.GetFrameIdx();
```

Comment:
Probably don't want to store the current frame index in the data. Doesn't make sense to me.

Reply:
I need the data in the other helper functions as well, and they can't access the render_system. This seemed better than modifying those functions to accept the render system as a variable and calling the GetFrameIdx function each time.